docs/api/cloudlink/intro.md

```diff
@@ -8,4 +8,9 @@
     - The general request/response shape
 -->
 
-This section is out of date.
+- This page is out of date.
+
+Up to date pages:
```

Comment:
i don't really like this approach and again, something should update these

Reply:
Makes sense, I will update that.

Reply:
I will update this when I get to it, just don't have time at the moment, do you have an idea on how the message should be written @williamhorning ?

Reply:
Actually, it seems as though most the only thing not up to date would be the authpswd page, which we could just add a notice box to